scripts/lib/check_rabbitmq_queue.py

```diff
@@ -155,6 +163,11 @@ def check_rabbitmq_queues() -> None:
         text=True,
     ).strip()
     queue_stats: Dict[str, Dict[str, Any]] = {}
+    if mobile_notification_shards > 1:
+        normal_queues += [
+            f"missedmessage_mobile_notifications_shard{d}"
+            for d in range(1, mobile_notification_shards + 1)
+        ]
```

Comment:
Hmm, will this still have the unshared mobile worker checked? I guess maybe that's OK?

Reply:
Only if it has consumers

Reply:
OK that seems fine.